hs_core/models.py

```diff
@@ -47,6 +47,50 @@
 from .languages_iso import languages as iso_languages
 
 
+class StorageCodes(object):
+    """
+    Storage codes describe what file storage medium is in use.
+    Storage is a numeric code 1-4:
```

Comment:
What is 4?  Is this a typo?

Reply:
looks like there was copied from an access control class and not updated completely for storage codes.